tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-frontend-test-ci-pipeline.yml

```diff
@@ -6,7 +6,7 @@ trigger: none
 jobs:
 - job: Onnxruntime_Linux_GPU_Training_FrontEnd
 
-  timeoutInMinutes: 120
+  timeoutInMinutes: 240
```

Comment:
>240 [](start = 20, length = 3)

you need 4 hours?

Reply:
each multiple choice finetune takes more than an hour.